test/integration/sip_test.go

```diff
@@ -376,10 +390,53 @@ func TestSIPJoinOpenRoom(t *testing.T) {
 		return dtmfIn == "4567"
 	}, 5*time.Second, time.Second/2)
 
+	go func() {
+		// TransferSIPParticipant is synchronous
+		_, err = lk.SIP.TransferSIPParticipant(context.Background(), &livekit.TransferSIPParticipantRequest{
+			RoomName:            "test-open",
+			ParticipantIdentity: "sip_" + clientNumber,
+			TransferTo:          "tel:" + transferNumber,
```

Comment:
Would be nice to also see if we can properly refer for outbound. But that can be done separately.